### PR TITLE
Expose PnP Problems on progress upload

### PR DESCRIPTION
**Pull request by CarsonF**
Screenshot 2024-08-28 at 12 54 20 PM
Screenshot 2024-08-28 at 12 54 35 PM

SeedCompany/cord-api-v3#3281

### CODE REVIEW

**Review by rdonigian — APPROVED** [review]
LGTM

File: src/components/DefinedFileCard/DefinedFileCard.tsx [hunk 1]
```diff
@@ -30,67 +31,9 @@ import { HugeIcon, ReportIcon } from '../Icons';
...
-    position: 'absolute',
-    right: 0,
-    top: 0,
-  },
-  text: {
-    marginTop: spacing(1),
-    textTransform: 'none',
-  },
-}));
-
```
Comment (andrewmurraydavid) [inline src/components/DefinedFileCard/DefinedFileCard.tsx:91]: I am SO grateful this is cleaned up!

File: src/scenes/ProgressReports/Detail/ProgressReportCard.tsx [hunk 2]
```diff
@@ -16,22 +32,102 @@ interface Props
...
+                    flag="pnp-validation"
+                    match={true}
+                    sx={{
+                      display: 'inherit',
+                      flexDirection: 'inherit',
+                      gap: 'inherit',
+                    }}
+                  >
+                    {reextracting ? (
+                      <CircularProgress size={15} sx={{ ml: 1.1 }} />
```
Comment (andrewmurraydavid) [inline src/scenes/ProgressReports/Detail/ProgressReportCard.tsx:93]: `ml: 1.1` ... I get it, but ugh... I which it was cleaner. Nit pick only

File: src/scenes/ProgressReports/Detail/ProgressReportCard.tsx [hunk 3]
```diff
@@ -16,22 +32,102 @@ interface Props
...
+          }
+          disableIcon
+          disablePreview
+          sx={{
+            width: file && 'max-content',
+          }}
           {...rest}
         />
-      </Tooltip>
+      </>
```
Comment (andrewmurraydavid) [inline src/scenes/ProgressReports/Detail/ProgressReportCard.tsx:120]: Not needed.

**Review by andrewmurraydavid — APPROVED** [review]
Looks good! Great work, @CarsonF !
Just some nitpicks, but can move forward without them.

File: src/scenes/ProgressReports/Detail/ProgressReportCard.tsx (hunk 2 above)
Comment (CarsonF) [reply src/scenes/ProgressReports/Detail/ProgressReportCard.tsx:93]: Yeah it was quick a dirty. Obviously we want the spinner centered in the spot where the button would be. But it has it's own specific sizing.
We could enhance our IconButton wrapper later to support this kinda thing.

File: src/scenes/ProgressReports/Detail/ProgressReportCard.tsx (hunk 3 above)
Comment (CarsonF) [reply src/scenes/ProgressReports/Detail/ProgressReportCard.tsx:120]: It was a quick disable that kept indentation. Revisit soon :tm:

File: src/scenes/ProgressReports/Detail/ProgressReportCard.tsx (hunk 2 above)
Comment (andrewmurraydavid) [reply src/scenes/ProgressReports/Detail/ProgressReportCard.tsx:93]: Yup. I get that. Had issues with CircularProgress myself in the past..

**Pull request merged**